src/presentation/http/router/note.test.ts

```diff
@@ -385,6 +385,105 @@ describe('Note API', () => {
     test.todo('Returns 400 when parentId has incorrect characters and lenght');
   });
 
+  describe('DELETE /note/:notePublicId', () => {
+    test('Returns 200 status and "true" if note was removed successfully', async () => {
+      const accessToken = global.auth(1);
+      const correctID = 'Pq1T9vc23Q';
+
+      let response = await global.api?.fakeRequest({
+        method: 'DELETE',
+        headers: {
+          authorization: `Bearer ${accessToken}`,
+        },
+        url: `/note/${correctID}`,
+      });
+
+      expect(response?.statusCode).toBe(200);
+
+      expect(response?.json()).toStrictEqual({ isDeleted: true });
+
+      response = await global.api?.fakeRequest({
+        method: 'DELETE',
```

Comment:
It's better to send GET note request to assert whether note was actually deleted